tests/customer-resource-show-visibility-test.js

```diff
@@ -98,12 +184,76 @@ describeApplication('CustomerResourceShowVisibility', () => {
       });
     });
 
-    it('displays the hidden/reason section', () => {
-      expect(CustomerResourceShowPage.isHidden).to.be.true;
+    it('the hide/show toggle with hide', () => {
+      expect(CustomerResourceShowPage.isHidden).to.be.false;
+    });
+
+    it('maps the hidden reason text', () => {
+      expect(CustomerResourceShowPage.hiddenReason).to.equal('');
+    });
+
+    describe('successfully showing a customer resource', () => {
+      beforeEach(function () {
+        this.server.timing = 50;
+        return CustomerResourceShowPage.toggleIsHidden();
+      });
+
+      afterEach(function () {
+        this.server.timing = 0;
+      });
+
+      it('reflects the desired state (Show)', () => {
+        expect(CustomerResourceShowPage.isHidden).to.be.false;
+      });
+
+      it('cannot be interacted with while the request is in flight', () => {
+        expect(CustomerResourceShowPage.isHiddenToggleable).to.be.false;
+      });
+
+      describe('when the request succeeds', () => {
+        it('reflect the desired state was set (Show)', () => {
+          expect(CustomerResourceShowPage.isHidden).to.be.false;
+        });
+
+        it('indicates it is no longer pending', () => {
+          expect(CustomerResourceShowPage.isHiding).to.be.false;
+        });
+      });
+    });
+  });
+
+  describe('visiting the customer resource show page and all titles in package are hidden', () => {
+    beforeEach(function () {
+      pkg.visibilityData.isHidden = true;
+      pkg.visibilityData.reason = 'Hidden by EP';
+      resource = this.server.create('customer-resource', 'isHidden', {
+        package: pkg,
+        title
+      });
+
+      let visibilityData = this.server.create('visibility-data', {
+        isHidden: true,
+        reason: ''
+      }).toJSON();
+
+      resource.update('visibilityData', visibilityData);
+      resource.save();
+
+      return this.visit(`/eholdings/customer-resources/${resource.id}`, () => {
+        expect(CustomerResourceShowPage.$root).to.exist;
+      });
+    });
+
+    it('the hide/show toggle with hide', () => {
```

Comment:
Alternate suggestion for this label: `displays the visibility toggle as switched to on`.